File: Dream.WebApp/ModelEntries/SecuritizationSummaryModelEntry.cs
namespace Dream.WebApp.ModelEntries
{
    public class SecuritizationSummaryModelEntry
    {
        public int SecuritizationNodeId { get; set; }
        public int? SecuritizationNodeTrancheId { get; set; }
        public string SecuritizationNodeName { get; set; }
        public string SecuritizationNodeOrTrancheName { get; set; }
        public string SecuritizationNodeOrTrancheType { get; set; }
        public string SecuritizationNodeOrTrancheRating { get; set; }
        public double? SecuritizationNodeOrTrancheBalance { get; set; }
        public double? SecuritizationNodeOrTrancheSizePercentage { get; set; }
        public double? SecuritizationNodeOrTrancheInitialCoupon { get; set; }
        public double WeightedAverageLife { get; set; }
        public double ModifiedDuration { get; set; }
        public string PaymentWindow { get; set; }
        public double? BenchmarkYield { get; set; }
        public double? NominalSpread { get; set; }
        public double InternalRateOfReturn { get; set; }
        public double PresentValue { get; set; }
        public double? DollarPrice { get; set; }
        public double PercentageOfCollateral { get; set; }
        public bool IsSecuritizationNode { get; set; }
    }
}

File: Dream.WebApp/app/classes/modelEntries/securitizationSummaryModelEntry.ts
export class SecuritizationSummaryModelEntry {  
    SecuritizationNodeId: number
    SecuritizationNodeTrancheId: number
    SecuritizationNodeName: string;
    SecuritizationNodeOrTrancheName: string;   
    SecuritizationNodeOrTrancheType: string;
    SecuritizationNodeOrTrancheRating: string;
    SecuritizationNodeOrTrancheBalance: number;
    SecuritizationNodeOrTrancheSizePercentage: number;
    SecuritizationNodeOrTrancheInitialCoupon: number;
    WeightedAverageLife: number;
    ModifiedDuration: number;
    PaymentWindow: string;
    BenchmarkYield: number;
    NominalSpread: number;
    InternalRateOfReturn: number;
    PresentValue: number;
    DollarPrice: number;
    PercentageOfCollateral: number;
    IsSecuritizationNode: boolean;
}